prodactivity/helpers/FetchHelper.ts

```diff
@@ -0,0 +1,115 @@
+// INTERFACES
+interface GetParams {
+    [index: string]: any
+}
+
+interface JSONResponse {
+    [index: string]: any
+}
```

Comment:
These interfaces should go in a separate file to export so we can use the type in other parameters.